backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  submissionDate               DateTime?                     @map("submission_date") @db.Timestamptz(6)
+  // if this field is true then the application is a confirmed duplicate
+  // meaning that the record in the application flagged set table has a status of duplicate
+  markedAsDuplicate            Boolean                       @default(false) @map("marked_as_duplicate")
+  confirmationCode             String                        @map("confirmation_code")
+  reviewStatus                 ApplicationReviewStatusEnum   @default(valid) @map("review_status")
+  userId                       String?                       @map("user_id") @db.Uuid
+  listingId                    String?                       @map("listing_id") @db.Uuid
+  applicantId                  String?                       @unique() @map("applicant_id") @db.Uuid
+  mailingAddressId             String?                       @unique() @map("mailing_address_id") @db.Uuid
+  alternateAddressId           String?                       @unique() @map("alternate_address_id") @db.Uuid
+  alternateContactId           String?                       @unique() @map("alternate_contact_id") @db.Uuid
+  accessibilityId              String?                       @unique() @map("accessibility_id") @db.Uuid
+  demographicsId               String?                       @unique() @map("demographics_id") @db.Uuid
+  applicationFlaggedSet        ApplicationFlaggedSet[]
+  applicant                    Applicant?                    @relation(fields: [applicantId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  accessibility                Accessibility?                @relation(fields: [accessibilityId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  alternateContact             AlternateContact?             @relation(fields: [alternateContactId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicationsAlternateAddress Address?                      @relation("applications_alternate_address", fields: [alternateAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  userAccounts                 UserAccounts?                 @relation(fields: [userId], references: [id], onUpdate: NoAction)
+  applicationsMailingAddress   Address?                      @relation("applications_mailing_address", fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                     listings?                     @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  demographics                 Demographics?                 @relation(fields: [demographicsId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  preferredUnitTypes           UnitTypes[]
+  householdMember              HouseholdMember[]
+
+  @@unique([listingId, confirmationCode])
+  @@index([listingId])
+  @@map("applications")
+}
+
+// Note: [file_id, label] formerly max length 128
+model Assets {
```

Comment:
Is this meant to be singular?

Reply:
https://github.com/bloom-housing/bloom/pull/3429#discussion_r1199453550

good question, establishing a good convention for this is defo a good idea